server/app/services/task-service.js

```diff
@@ -0,0 +1,173 @@
+const logger = require('../../logger');
+const { Barman, Kommission, Task } = require('../models');
+const { createUserError, createServerError, cleanObject, setEmbeddedAssociations } = require('../../utils');
+const sequelize = require('../../db');
+
+/**
+ * Create a task.
+ *
+ * @param newTask {Task} partial task
+ * @param _embedded {Object} Object containing associations to update, see swagger for more information.
+ * @return {Promise<Task|Errors.ValidationError>} The created task with its id
+ */
+async function createTask(newTask, _embedded) {
+
+    logger.verbose('Task service: creating a new task named %s', newTask.name);
+
+    const transaction = await sequelize.transaction();
+    try {
+        newTask.kommissionId = _embedded.kommissionId;
```

Comment:
Here we can check if the kommission exists (anyway if the komm does not exist, request will fail, but we with a server error)

Reply:
Yes